ClientApp/src/Pages/WhaleSpecies/WhaleSpecies.tsx

```diff
@@ -1,12 +1,53 @@
 import "./WhaleSpecies.scss";
 import "bootstrap";
 import React from "react";
-import {WhaleData} from "./WhaleSpeciesData";
+import { WhaleCardData, WhaleData } from "./WhaleSpeciesData";
 
-export function WhaleSpecies(): JSX.Element {
-  
+
+function Photo(data: WhaleCardData): JSX.Element {
     return (
-        <p>To be written</p>
-    );
+        <li className="carousel__slide">
+            <figure className="small-screen">
+                <div>
+                    <img src={data.imgSrc} alt={data.title} />
+                </div>
+                <figcaption>
+                    <h5 className="card-title sub-heading" >{data.title}</h5>
+                    <p className="photo-text">{data.text}</p>
+                    <a href={data.siteLink} className="btn btn-primary">
+                        Read more
+                </a>
+                </figcaption>
+            </figure>
+        </li>
+    )
+}
 
+export function WhaleSpecies(): JSX.Element {
+    return (
+        <section className="section content-container" >
+            <div className="container" >
+                <h1 className="title">Whale species</h1>
+                <div className="carousel" >
+                    <input type="radio" name="slides" checked id="slide-1" />
+                    <input type="radio" name="slides" id="slide-2" />
+                    <input type="radio" name="slides" id="slide-3" />
+                    <ul className="carousel__slides">
+                        {WhaleData.map(x => Photo(x))}
+                    </ul>
+                    <ul className="carousel__thumbnails">
+                        <li>
+                            <label htmlFor="slide-1"><img src={WhaleData[0].imgSrc} alt={WhaleData[0].title} /></label>
+                        </li>
+                        <li>
+                            <label htmlFor="slide-2"><img src={WhaleData[1].imgSrc} alt={WhaleData[1].title} /></label>
+                        </li>
+                        <li>
+                            <label htmlFor="slide-3"><img src={WhaleData[2].imgSrc} alt={WhaleData[2].title} /></label>
+                        </li>
```

Comment:
could we refactor this to loop over WhaleData and create a li element and label/img element for each piece of data? It'll mean if we ever add data to this we won't need to change this code :)